tests/integration/cattletest/core/common_fixtures.py

```diff
@@ -33,13 +37,42 @@ def _client_for_user(name, accounts):
                            secret_key=accounts[name][1])
 
 
+def client_for_project(project):
+    access_key = random_str()
+    secret_key = random_str()
+    admin_client = _admin_client()
+    active_cred = None
+    account = project
+    for cred in account.credentials():
```

Comment:
I'm a little confused.  Does this loop do anything?  access_key and secret_key are always declared a random string prior to this loop.  I know we handle active_cred is None below, but it seems like that will always be the case.